src/services/aws/wise.ts

```diff
@@ -29,8 +30,10 @@ export const wise = aws.injectEndpoints({
     recipient: builder.query<V2RecipientAccount, string>({
       query: (id: string) => `${baseURL}/v2/accounts/${id}`,
     }),
-    currencis: builder.query<WiseCurrency[], unknown>({
+    wiseCurrencies: builder.query<Currency[], unknown>({
       query: () => `${baseURL}/v1/currencies`,
+      transformResponse: (res: WiseCurrency[]) =>
+        res.map((r) => ({ rate: null, code: r.code, name: r.name })),
```

Comment:
```suggestion
        res.map((r) => ({ code: r.code, name: r.name })),
```
We can update the type of `Currency.rate` to be `rate?: number`, so there would be no need to set it here.

Reply:
see [related comment](https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2709#discussion_r1482357610) - having `null` here isn't critical (a bit helpful, tells otherwise that wise currencies no need to utilize `rate` property ) that we need to `rate` optional across.

here's a [youtube video](https://www.youtube.com/watch?v=X_tvQoW4ioI) that expounds this nuance

Reply:
I watched the video, the creator recommends the use of union with undefined (or null in this case) arguments only if there's an explicit use case for it; he goes on to give an example that's similar to our `ErrorContext.handleError`:
```ts
function logError(error: Error | undefined, message: string) { /** ... */ }
```
In all other cases even he recommends the use of optional arguments (`?`)